pkg/operator/ceph/csi/spec.go

```diff
@@ -259,6 +261,17 @@ const (
 	NFSDriverShortName    = "nfs"
 	rbdDriverSuffix       = "rbd.csi.ceph.com"
 	cephFSDriverSuffix    = "cephfs.csi.ceph.com"
+
+	// default crush location labels.
+	defaultCrushLocationLabels = "topology.kubernetes.io/region," +
```

Comment:
> I wonder if there is an easy way to generate this comma-separated list directly from `CRUSHTopologyLabels` array to avoid duplication.

Tried to but we'll hit cyclic imports error, so there's no simple way to do it for now.

Reply:
How about moving the definition of `CRUSHTopologyLabels` somewhere where it can be easily imported from?

Reply:
Moved around stuff, its working now.